package.json

```diff
@@ -46,5 +47,6 @@
         "rollup-plugin-add-shebang": "^0.3.1",
         "rollup-plugin-preserve-shebangs": "^0.2.0",
         "typescript": "^5.4.5"
-    }
+    },
+    "packageManager": "pnpm@9.3.0"
```

Comment:
Added this field as the pnpm action will auto-install the version that's specified here (see: https://github.com/svelte-add/svelte-add/pull/379#discussion_r1633657080).